Gemfile

```diff
@@ -2,9 +2,11 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'ransack-mongoid', git: 'https://github.com/activerecord-hackery/ransack-mongoid'
+# gem 'ransack-mongoid', git: 'https://github.com/activerecord-hackery/ransack-mongoid'
+gem 'ransack-mongoid', github: 'jesusalc/ransack-mongoid', branch: 'rails-6-mongoid-7-ruby-2-7-0'
```

Comment:
Should we first propose these changes to ransack-mongoid and discuss them there?

Reply:
@jesusalc could you open a PR to ransack-mongoid? That's actually a better idea 😉